company/create.js

```diff
@@ -1,46 +1,22 @@
 import { v1 as uuidv1 } from 'uuid';
 import * as dynamoDbLib from '../libs/dynamodb-lib';
-import { failure, success, validationError } from '../libs/response-lib';
-import { companySchema } from '../validation/companySchema';
 
 export const main = async event => {
-  /** @type {import('../typings/company).Company} */
-  const data = JSON.parse(event.body);
-
-  try {
-    await companySchema('required').validateAsync(data, { abortEarly: false });
-  } catch (e) {
-    console.log(e);
-    const errorMessages = e.details.map(detail => detail.message);
-    return validationError(errorMessages);
-  }
+  const { sub, email } = event.request.userAttributes;
 
   const params = {
     TableName: process.env.COMPANIES_TABLE_NAME,
     Item: {
       companyId: uuidv1(),
-      acceptedTerms: data.acceptedTerms,
-      companyName: data.companyName,
-      copyAcceptedTerms: data.copyAcceptedTerms,
-      email: data.email,
-      firstName: data.contactFirstName,
-      lastName: data.companyLastName,
-      iban: data.iban,
-      kvk: data.kvk,
-      city: data.address.city,
-      houseNumber: data.address.houseNumber,
-      street: data.address.street,
-      zipCode: data.address.zipCode,
-      latitude: data.address.latitude,
-      longitude: data.address.longitude
+      userId: sub,
+      email: email
     }
   };
 
   try {
     await dynamoDbLib.call('put', params);
-    return success(params.Item);
+    return event;
   } catch (e) {
-    console.log(e);
-    return failure({ status: false });
+    return event;
   }
```

Comment:
You can clean this bit even more:
```javascript
try {
  await dynamoDbLib.call('put', params);
} catch (e) {
  console.error(e);
}

return event
```

Reply:
I can't because cognito expects to return event either is success or failure

Reply:
oh sorry,
good point